app/controllers/partners/individuals_requests_controller.rb

```diff
@@ -2,9 +2,11 @@ module Partners
   class IndividualsRequestsController < BaseController
     def new
       @request = FamilyRequest.new({}, initial_items: 1)
-      @requestable_items = Organization.find(current_partner.diaper_bank_id).valid_items.map do |item|
-        [item[:name], item[:id]]
-      end.sort
+
+      requestable_items = PartnerFetchRequestableItemsService.new(partner_id: current_partner.partner.id).call
+      @formatted_requestable_items = requestable_items.map do |rt|
+        [rt.name, rt.id]
+      end
```

Comment:
now that I'm seeing the same block here and other controllers, maybe we should have a "presenter" object to do the formatting. Arguably this duplication existed before this PR, so we could do it as a follow up too

Reply:
Yeh I agree, I think some sort of mechanism to avoid duplication would be good! Will create an issue to handle that in another PR.